rllib/examples/documentation/replay_buffer_demo.py

```diff
@@ -51,32 +50,31 @@
 
 
 # __sphinx_doc_replay_buffer_own_buffer__begin__
-class LessSampledReplayBuffer(ReplayBuffer):
+class SimpleMixInBuffer(ReplayBuffer):
     @override(ReplayBuffer)
     def sample(
         self, num_items: int, evict_sampled_more_then: int = 30, **kwargs
     ) -> Optional[SampleBatchType]:
-        """Evicts experiences that have been sampled > evict_sampled_more_then times."""
-        idxes = [random.randint(0, len(self) - 1) for _ in range(num_items)]
-        often_sampled_idxes = list(
-            filter(lambda x: self._hit_count[x] >= evict_sampled_more_then, set(idxes))
-        )
-
-        sample = self._encode_sample(idxes)
+        """At least 1/2 of samples is always newest samples.
```

Comment:
The "LessSampled" only served to illustrate but would need more code than this example should contain with the storage API.